.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min'
```

Comment:
Doesn't this work: 
```suggestion
import { MemoryRouter } from 'react-router-dom'
```

Reply:
Yes, it does. The link came automatic after I did the import